flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -1596,4 +1597,83 @@ private Stream<TestSetSpec> arraySortTestCases() {
                                 },
                                 DataTypes.ARRAY(DataTypes.DATE())));
     }
+
+    private Stream<TestSetSpec> splitTestCases() {
+        return Stream.of(
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.SPLIT)
+                        .onFieldsWithData(
```

Comment:
Please include tests also for
- SPLIT("", "")
- SPLIT("", ",")
- SPLIT(",,", ",,")
- - SPLIT(",,", ",,,")
- SPLIT("sssss", "ss")